docker/daemon_unix.go

```diff
@@ -11,7 +11,6 @@ import (
 	"github.com/docker/docker/daemon"
 	"github.com/docker/docker/pkg/system"
 
-	_ "github.com/docker/docker/daemon/execdriver/lxc"
```

Comment:
It that really correct - removing the lxc driver on *nix? Not sure, but looks a little odd being in this PR.